lib/fetchPosts.tsx

```diff
@@ -1,48 +1,63 @@
-import { FeedType } from '@/providers/FeedProvider';
 import { SupabasePost } from '@/types/SupabasePost';
 import { SupabaseClient } from '@supabase/supabase-js';
+import getFollowing from './neynar/getFollowing';
+import { fetchPostsLimit } from './consts';
+import { FeedType } from '@/types/Feed';
 
-const fetchPosts = async (
+const getBaseQuery = (
```

Comment:
please move `getBaseQuery` to it's own file

each file should do exacly one thing
- actual: 2 functions defined
- expected: only one function defined per file